packages/clerk-js/src/ui/components/Checkout/CheckoutPage.tsx

```
@@ -6,125 +1,73 @@
-    <CheckoutContextRoot.Provider
-      value={{
-        checkout,
-        isLoading: isMutating,
-        updateCheckout,
-        errors,
-        startCheckout,
-        status,
-      }}
+    <CheckoutProvider
+      for={subscriberType === 'org' ? 'organization' : undefined}
+      planId={planId!}
```

ESLint

[error] Forbidden non-null assertion.

(@typescript-eslint/no-non-null-assertion)